boards/common/stm32/dist/stm32l0.cfg

```diff
@@ -1,3 +1,8 @@
-source [find target/stm32l0.cfg]
+# Try to use dual bank if available and supported
+try {
+    source [find target/stm32l0_dual_bank.cfg]
+} on error {} {
+    source [find target/stm32l0.cfg]
```

Comment:
Here I would replace it with a nice error message "you need an openocd version that supports dual bank (which one ?) and maybe a pointer to https://github.com/gnu-mcu-eclipse/openocd/releases.

Reply:
No, because it's the fallback version that should work if dual_bank.cfg is not available (typically the case with openocd 0.10 stable version).
If dual_bank is really required, OpenOCD will fail later in the flash workflow.

Reply:
It will fail with an unrelated error message. I do not want to answer issues on "why is it not working on my machine" and I do not want to verify this is still working at each realeases.

If you do, then at least put a comment that it is a hack to fallback but that the RIOT generated files may not work with this configuration.

Then explain to users trying to run all tests on their toolchain and saying `riotboot` is broken.

Reply:
+1 for not explicitly removing the single slot support. We're effectively preventing users from flashing smaller binaries for this board, if they have the openocd stable version. I would point out that this will also lead to questions about why it's suddenly stopped working, for certain users.

Suggest keeping `source [find target/stm32l0.cfg]` and giving them an info message, which also declares that support for this toolchain for this board is deprecated (i.e. we don't do support activities) and that they should upgrade if they want to use the dual bank functionality.